src/pages/Transportation/Register/SacredShrines.tsx

```diff
@@ -1,3 +1,92 @@
+import { Header } from "../../../components/header";
+import noble1 from "../../../assets/Transportation/register/noble_1.webp";
+import noble2 from "../../../assets/Transportation/register/noble_2.webp";
+import noble3 from "../../../assets/Transportation/register/noble_3.webp";
+import noble4 from "../../../assets/Transportation/register/noble_4.webp";
+import noble5 from "../../../assets/Transportation/register/noble_5.webp";
+import noble6 from "../../../assets/Transportation/register/noble_6.webp";
+
+import { RegisterCard } from "../../../components/Rigester_card";
+
 export function SacredShrines() {
-  return <div>SacredShrine</div>;
+  return <>
+  <Header heading="Groups of nobles" img="headerBg" />
+  <section className="">
+
+    <div className="flex gap-5 mb-8">
```

Comment:
please make an array of objects for the data. like

const scaredShrineData = [
 {
   heading: "add heading",
   img: "add image",
   description: "details about card",
    date: " ",
    date1: " ",
    date2: " ",
    price: " ",
    rate: " ",
    button: " ",
}
]


and then map this array render the RegisterCard component.